modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -806,6 +807,10 @@ If you need to debug the import, it might be useful to collect the stack trace.
 This is done by using the `--verbose` option.
 |import.report
 
+|--schema=<path> footnote:[The optional `--schema` option is available in this version but not yet supported. It will be functional in a future release.]
```

Comment:
I might be picky. But is it clear when we say 'the ... option is available but not supported'? 
Could we just say that this option doesn't work with the incremental import command? Or - 'The incremental import command currently doesn't support the `--schema` option. It will be functional in a future release'?

Reply:
That would be the best, but I didn't want to say that. I wanted to say it more nicely 😄. If we say, "The incremental import command currently doesn't support the—-schema option," is that not true because we display it? I would say that incremental import supports it, but it doesn't work 😄 😄. It's a tricky one. I am open to more variations.

Reply:
what do you think about:

The `--schema` option is available in this version but is not yet supported. It will be functional in a future release.

Reply:
Yes, we don't need the "optional" part.